packages/salesforcedx-vscode-apex-debugger/package.json

```diff
@@ -15,7 +15,101 @@
   "engines": {
     "vscode": "^1.13.0"
   },
-  "categories": [
-    "Debuggers"
-  ]
+  "categories": ["Debuggers"],
+  "dependencies": {
+    "@salesforce/salesforcedx-utils-vscode": "40.4.0",
+    "vscode-debugprotocol": "1.19.0",
+    "vscode-debugadapter": "1.19.0"
+  },
+  "devDependencies": {
+    "@types/chai": "^4.0.0",
+    "@types/mocha": "2.2.38",
+    "@types/node": "^6.0.40",
+    "@types/sinon": "1.16.31",
+    "chai": "^4.0.2",
+    "mocha": "3.2.0",
+    "sinon": "1.17.4",
```

Comment:
This version of sinon seems different from the ones in other packages.
Can we use the same ones across all packages?

I only noticed this because I was also checking the sinon version earlier for the stuff I was working on.